### ISSUE TITLE
Enhancement request : +/- 10, 100, 1000, etc buttons for frequency and mouse wheel modification of frequency number

### ISSUE THREAD

**Issue by sblandford** [PhantomSDR/PhantomSDR#8]
It's difficult to set the frequency to an exact step just by sliding the tuner. Some way of nudging it up or down to a rounded frequency using buttons and/or adjusting the digits for the frequency with the mouse scroll wheel would be cool.

Currently the only option to get an exact frequency I can find is to click and on the number and type/edit it.

**Comment by rhgndf** [comment]
Commit efe7a4a brings in the updated frontend with a frequency input control similar to other SDR softwares.

**Comment by sblandford** [comment]
Thanks! Just tried it and it works well.

**Comment by sblandford** [comment]
Apologies for re-opening this issue.

The scroll wheel works really well on the desktop however getting the frequency exactly right on a mobile device is still a bit of a challenge. Would it be possible to also implement step buttons? The 1KHz steps zeroing the 100s, 10s and 1s Hz is a nice feature because most things I listen to are usually exactly on a 1KHz step. So this would be good on the step buttons also. Here is one possible layout...

![image](https://github.com/PhantomSDR/PhantomSDR/assets/4608191/997268d5-a58c-41ca-8039-b6f6b522aa34)

In this example, the 1KHz steps are slightly emphasised to indicate their "zeroing" functionality.

**Comment by rhgndf** [comment]
There is already something implemented: a tap and hold method of tuning. Tap on the digit and drag it up or down to do the tuning.

**Comment by sblandford** [comment]
Yes, I just checked the tap-and-hold on a mobile and it works, thanks. Although it definitely needs documenting and casual visitors to a WebSDR are unlikely to read the documentation.

**Comment by sblandford** [comment]
Great thanks!

I just checked it and it certainly improves usability on mobile devices. It would be nice if the +1K and -1K buttons zeroed the .000 Hz values similar to the scroll wheel action.

Also, I noticed that on my Pixel 6 phone the control panel is cropped below the Frequency Offset line in Portrait mode. In Landscape mode, it looks OK.
![image](https://github.com/PhantomSDR/PhantomSDR/assets/4608191/445c29b7-7a29-4378-9810-ee9bd9424a3f)
